CHANGELOG.md

```diff
@@ -12,16 +12,27 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.1.0/)
 ### Maintenance
 ### Refactoring
 
-## [Unreleased 2.x](https://github.com/opensearch-project/flow-framework/compare/2.14...2.x)
+## [Unreleased 2.x](https://github.com/opensearch-project/flow-framework/compare/2.13...2.x)
 ### Features
 ### Enhancements
-- Add Workflow Step for Reindex from source index to destination ([#718](https://github.com/opensearch-project/flow-framework/pull/718))
-- Add param to delete workflow API to clear status even if resources exist ([#719](https://github.com/opensearch-project/flow-framework/pull/719))
+- Add guardrails to default use case params ([#658](https://github.com/opensearch-project/flow-framework/pull/658))
+- Allow strings for boolean workflow step parameters ([#671](https://github.com/opensearch-project/flow-framework/pull/671))
+- Add optional delay parameter to no-op step ([#674](https://github.com/opensearch-project/flow-framework/pull/674))
+- Add model interface support for remote and local custom models ([#701](https://github.com/opensearch-project/flow-framework/pull/701))
 
 ### Bug Fixes
-- Add user mapping to Workflow State index ([#705](https://github.com/opensearch-project/flow-framework/pull/705))
+- Reset workflow state to initial state after successful deprovision ([#635](https://github.com/opensearch-project/flow-framework/pull/635))
+- Silently ignore content on APIs that don't require it ([#639](https://github.com/opensearch-project/flow-framework/pull/639))
+- Hide user and credential field from search response ([#680](https://github.com/opensearch-project/flow-framework/pull/680))
+- Throw the correct error message in status API for WorkflowSteps ([#676](https://github.com/opensearch-project/flow-framework/pull/676))
+- Delete workflow state when template is deleted and no resources exist ([#689](https://github.com/opensearch-project/flow-framework/pull/689))
+- Fixing model group parsing and restoring context ([#695] (https://github.com/opensearch-project/flow-framework/pull/695))
 
 ### Infrastructure
+- Switch macos runner to macos-13 from macos-latest since macos-latest is now arm64 ([#686](https://github.com/opensearch-project/flow-framework/pull/686))
+
 ### Documentation
 ### Maintenance
 ### Refactoring
+- Improve error messages for workflow states other than NOT_STARTED ([#642](https://github.com/opensearch-project/flow-framework/pull/642))
+- Create a Config XContent model for Config index ([#679](https://github.com/opensearch-project/flow-framework/pull/679))
```

Comment:
Eh! Looks like this file has other commits now. I would suggest to not add any entry and not change this file because this a util method. We generally add feature related enhancement here.

Reply:
okay i will not change the changelog at all

Reply:
Now we can see your entry which should be fine. Let's wait for all the checks to pass

Reply:
okay!